### ISSUE TITLE
Chached chat-log instead of google add?

### ISSUE THREAD

**Issue by poVoq** [arjunsuresh/chat#1]
Would it be possible to display a cached chat-log for non-logged in users instead of the google ad?

How is compatibility of this with Q2A 1.8 by the way?

**Comment by arjunsuresh** [comment]
Do you mean an image for the chat log? That can be done.

**Comment by poVoq** [comment]
No. Although that might also be okish.

I mean that there is a static but scrollable version of the chat that gets updated every 30 minutes or so (with messages from the last few hours/days), that is served to non-logged in users with a message log in to start chatting.

This:
https://github.com/q2a-projects/Q2A-Ultimate-Widgets
seems to have such a kind of widget caching system for RSS feeds for example.

**Comment by arjunsuresh** [comment]
oh okay. There must be a way as Ajax chat can be configured for anonymous users. Personally I'm not interested in that unless one runs a dedicated server just for chat. The load caused by even logged in users are quite high on the server -- why increase it with anonymous users too :)

**Comment by poVoq** [comment]
No, I don't want anonymous users either... too much abuse and spam.
I just want non-logged in users be able to read the latest messages.
